metricbeat/mb/testing/data/data_test.go

```diff
@@ -83,13 +112,18 @@ func TestAll(t *testing.T) {
 			config.Suffix = "json"
 		}
 
-		getTestdataFiles(t, config.URL, moduleName, metricSetName, config.Suffix)
+		getTestdataFiles(t, moduleName, metricSetName, config)
 	}
 }
 
-func getTestdataFiles(t *testing.T, url, module, metricSet, suffix string) {
+func getTestdataFiles(t *testing.T, module, metricSet string, config Config) {
+	if *moduleFlag != "" {
```

Comment:
I'm now thinking if we should even move this to line 100? Like this we can skip even earlier.

Reply:
Moved